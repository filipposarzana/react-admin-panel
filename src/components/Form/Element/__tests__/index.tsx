import { createRenderer } from 'react-test-renderer/shallow'
import { Form } from '..'

describe('Form', () => {
  it('matches snapshot', () => {
    const renderer = createRenderer()

    renderer.render(<Form />)

    const output = renderer.getRenderOutput()

    expect(output).toMatchSnapshot()
  })
})
