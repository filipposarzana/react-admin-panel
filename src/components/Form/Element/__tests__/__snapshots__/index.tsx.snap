// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form matches snapshot 1`] = `
<form
  className="sc-19qvh5f-0 sc-1wc2hvr-0 ioBge"
  direction="column"
  noValidate={true}
  wrap="nowrap"
/>
`;
